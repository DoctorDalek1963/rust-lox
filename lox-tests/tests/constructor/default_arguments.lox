class Foo {}

var foo1 = Foo();
var foo2 = Foo(1, 2, 3); // expect runtime error: Expected 0 arguments but got 3.
