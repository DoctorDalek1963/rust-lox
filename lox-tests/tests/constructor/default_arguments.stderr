ERROR: Expected 0 arguments in function call but got 3
 --> 4:12
  |
4 | var foo2 = Foo(1, 2, 3); // expect runtime error: Expected 0 arguments but got 3.
  |            ^----------^
